server/internal/app/private.go

```diff
@@ -34,3 +34,25 @@ func ExportLayer() echo.HandlerFunc {
 		return c.Stream(http.StatusOK, mime, reader)
 	}
 }
+
+func ExportDataset() echo.HandlerFunc {
```

Comment:
Define this method to `adapter/http`. Usecases should be called on the adapter layer.

```
app -call-> adapter -call-> usecase -use-> domain
```